qiskit_ibm_runtime/utils/utils.py

```diff
@@ -29,38 +29,40 @@
 )
 from ibm_platform_services import ResourceControllerV2  # pylint: disable=import-error
 from qiskit.circuit import QuantumCircuit
-from qiskit.transpiler import Target
+from qiskit.providers.backend import BackendV2
 from qiskit_ibm_runtime.exceptions import IBMInputValueError
 
 
-def is_isa_circuit(circuit: QuantumCircuit, target: Target) -> bool:
+def is_isa_circuit(circuit: QuantumCircuit, backend: BackendV2) -> bool:
     """Checks if the circuit is an ISA circuit, meaning that it has a layout and that it
     only uses instructions that exist in the target.
+
     Args:
         circuit: A single QuantumCircuit
-        target: A Qiskit Target
+        backend: The target backend.
+
     Returns:
         Boolean True if the circuit is an ISA circuit
     """
-    if circuit.layout is None:
+    if backend.coupling_map and circuit.layout is None:
```

Comment:
I would probably just remove this check. Having the `layout` attribute set isn't hard requirement to run the circuit. Like you could manually build a hardware aware circuit and directly submit it and it would otherwise be completely valid to run right?